doc/nest-09.md

```diff
@@ -0,0 +1,55 @@
+# Managing Nest's repositories
+
+Several repositories are available for Nest. The official ones are `stable`, `beta` and `nightly`.
+
+When a package is uploaded, it first goes in the `nightly` repository. We (the Raven team and people we trust) go through all the packages and test them, to see if the installation/removal work as expected, and if we can't find any bugs. If everything's good, the package is moved to `beta`. We're not looking for bugs anymore at this point, but if you find any, don't hesitate to report them to the package owner. After a while, if no bug is being reported, the package is moved to `stable`.
```

Comment:
That's not entirely correct:
* A package in `nightly` is a package under construction where some parts may be missing or not working at all. 
* A package in `beta` is a package well-formed that isn't fully debbuged yet. A package stays here for a bit of time until no one reports bugs anymore.